apps/web/components/notification-sound-handler.tsx

```diff
@@ -0,0 +1,142 @@
+"use client";
+
+import { useEffect, useRef } from "react";
+
+export function NotificationSoundHandler() {
```

Comment:
This is required to load the audio as soon as user interacts with our site.

https://developer.chrome.com/blog/autoplay